casbin/README.md

```diff
@@ -153,3 +153,63 @@ func main() {
   app.Listen(":8080")
 }
 ```
+## Notes on Config's Enforcer
+
+Use this property instead, if you
```

Comment:
The wording here needs some work.

Reply:
you're free to modify to whatever you see fit. I'm just sharing what I've studied so far.